openprescribing/frontend/management/commands/measure_sql/normalised_prescribing_standard.sql

```diff
@@ -0,0 +1,26 @@
+SELECT
```

Comment:
Nit: the views in the current directory are no longer all measure-related, despite what the directory name implies. It would be nice to call it something like `views_sql` rather than `measures_sql`.

Reply:
It was already not completely measure-related.  The time to do this will be when we drop `import_measures` -- have got too many balls in the air right now.